### PR TITLE
Add the ability to plot striped lines

### PR DESCRIPTION
**Pull request by lucielavigne**
## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

**Comment by jklymak** [comment]
Can you add a PR summary or mark as draft if you are not ready for review yet?

File: lib/matplotlib/lines.py [hunk 1]
```diff
@@ -1046,6 +1064,23 @@ def set_color(self, color):
         self._color = color
         self.stale = True
 
+    def set_offcolor(self, offcolor):
+        """
+        Set the offcolor of the line.
+        By default set at 'None'
```
Comment (timhoffm) [inline lib/matplotlib/lines.py:1070]:
```suggestion
        Set the gap color for the dashed line style.
```

File: lib/matplotlib/lines.py [hunk 2]
```diff
@@ -1046,6 +1064,23 @@ def set_color(self, color):
...
+
+        Parameters
+        ----------
+        offcolor : color or None
+        """
```
Comment (timhoffm) [inline lib/matplotlib/lines.py:1075]:
```suggestion
        offcolor : color or None
            The gap color. If None, the gaps are unfilled.
        """
```

**Review by timhoffm — COMMENTED** [review]
Thanks for the PR, and welcome to Matplotlib!

Please fix the comments and flake8 issues.

**Comment by timhoffm** [comment]
@lucielavigne do you plan to continue working on this?

**Pull request closed without merging**